cmd/dev.go

```diff
@@ -224,14 +250,48 @@ var devCmd = &cobra.Command{
 	},
 }
 
+func checkHostnames() error {
+	hostnames := []string{compose.HostLocalDashboardNhostRun, compose.HostLocalGraphqlNhostRun, compose.HostLocalAuthNhostRun, compose.HostLocalStorageNhostRun, compose.HostLocalFunctionsNhostRun, compose.HostLocalMailNhostRun}
+
+	for _, hostname := range hostnames {
+		_, err := net.LookupIP(compose.HostLocalDashboardNhostRun)
+		if err != nil {
+			logger.DEBUG = false
+			fmt.Println(fmt.Sprintf(`Failed to resolve '%s' hostname
+
+Make sure that you have internet connection.
```

Comment:
Please make sure you have an internet connection and try again.
```suggestion
Make sure that you have internet connection.
```